### PR TITLE
Add Missing connection to database

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/investigator/issues/89

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Fix issue and move logic

### CODE REVIEW

File: thoth/investigator/investigate_unresolved_package.py [hunk 1]
```diff
@@ -88,7 +84,7 @@ def investigate_unresolved_package(file_test_path: Optional[Path] = None) -> Tup
     parameters = content["result"]["parameters"]
     runtime_environment = parameters["project"].get("runtime_environment")
 
-    solver = OPENSHIFT.obtain_solver_from_runtime_environment(runtime_environment=runtime_environment)
+    solver = OpenShift.obtain_solver_from_runtime_environment(runtime_environment=runtime_environment)
```
Comment (saisankargochhayat) [inline thoth/investigator/investigate_unresolved_package.py:87]: https://github.com/thoth-station/common/blob/3876c4e97a68001d242bc56490df55f56ab61f2c/thoth/common/openshift.py#L227
Seems to be a class method?

Comment (pacospace) [reply thoth/investigator/investigate_unresolved_package.py:87]: > https://github.com/thoth-station/common/blob/3876c4e97a68001d242bc56490df55f56ab61f2c/thoth/common/openshift.py#L227
> Seems to be a class method?

Yes, but you can run without initiating the class:

```
from thoth.common.openshift import OpenShift
OpenShift.parse_python_solver_name(solver_name="fgsds")`

Traceback (most recent call last):
  File "test_workflow_status.py", line 65, in <module>
    OpenShift.parse_python_solver_name(solver_name="fgsds")
  File "/home/fmurdaca/work/aicoe/common/thoth/common/openshift.py", line 202, in parse_python_solver_name
    f"Solver name has to start with 'solver-' prefix: {solver_name!r}"
thoth.common.exceptions.SolverNameParseError: Solver name has to start with 'solver-' prefix: 'fgsds'

```

Comment (saisankargochhayat) [reply thoth/investigator/investigate_unresolved_package.py:87]: Am I missing something?
```
(investigator) [sai@sai-pc investigator]$ python
Python 3.6.11 (default, Jul  1 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from thoth.common.openshift import OpenShift
>>> OpenShift.parse_python_solver_name(solver_name="solver-fgsds")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: parse_python_solver_name() missing 1 required positional argument: 'self'
```

Comment (pacospace) [reply thoth/investigator/investigate_unresolved_package.py:87]: what version of thoth-common?, v0.14.2 is the new one

**Pull request merged**